File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/anagrafiche/ParticelleCatastaliClassamento.cs
using AgronicaCoreModelsSTD.baseClass;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.anagrafiche
{
    public class ParticelleCatastaliClassamento
    {
        public string porzione { get; set; }
        public double Area { get; set; }
        public BaseCodeDescr qualita { get; set; }

        public string classe { get; set; }

        public double redditoDomiciliare { get; set; }
        public double redditoAgrario { get; set; }

        public ParticelleCatastaliClassamento()
        {

        }
    }
}

File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/anagrafiche/ParticelleCatastaliClassamento.ts
import { BaseCodeDescr } from '../baseClass/baseCodeDescr';
import { Macrouso } from '../metaschema/Macrouso';
import { IntervalloTemporale } from './IntervalloTemporale';

export class ParticelleCatastaliClassamento {
    porzione: string;
    Area: number;
    qualita: BaseCodeDescr;
    classe: string;
    redditoDomiciliare: number;
    redditoAgrario: number;

}
